projects/packages/videopress/changelog/update-videopress-button-focus-styles

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: changed
+
+VideoPress: tweak Edit button focus styles of the VideoDetailsCard component
```

Comment:
The focus style did not change, only the position of the dropdown

Reply:
Yes, initially the PR was a different purpose. Nice catcg. let me update it